schema_editor/app/scripts/schemas/schemas-service.js

```diff
@@ -0,0 +1,27 @@
+
+(function () {
+    'use strict';
+
+    /* ngInject */
+    function Schemas() {
+        var module = {
+            Object: Object
+        };
+        return module;
+
+        function Object() {
```

Comment:
Yeah, looks like jshint is complaining about this and a couple other things in this file. We should probably have Travis run jshint.